packages/utils/lib/environment/parse.ts

```diff
@@ -56,6 +56,7 @@ export const ENVS = z.object({
     IDLE_MAX_DURATION_MS: z.coerce.number().default(0),
     RUNNER_NODE_ID: z.coerce.number().optional(),
     RUNNER_URL: z.string().url().optional(),
+    RUNNER_MEMORY_WARNING_THRESHOLD: z.coerce.number().optional().default(85),
```

Comment:
increase the limit to 85 because I don't want to spam logs too much. We can tweak this value later